hera_cal/abscal_funcs.py

```diff
@@ -1113,6 +1112,66 @@ def interp2d_vis(model, model_lsts, model_freqs, data_lsts, data_freqs, flags=No
     return DataContainer(new_model), DataContainer(new_flags)
 
 
+def rephase_vis(model, model_lsts, data_lsts, bls, freqs, flags=None, max_dlst=0.005, latitude=-30.72152):
```

Comment:
`max_dlst = .005` corresponds to 69 seconds, which seems a bit long for a default. I'd bump it down to .002 or so.

Reply:
well, for a 14.6 m baseline I don't think its terribly unreasonable. But for a longer baseline it probably isn't great. we probably should expose it.